morebits.js

```diff
@@ -3221,12 +3222,13 @@ Morebits.wiki.page = function(pageName, status) {
 	 * @param {Function} onSuccess - Callback function to be called when
 	 * the username and timestamp are found within the callback.
 	 */
-	this.lookupCreation = function(onSuccess) {
+	this.lookupCreation = function(onSuccess, onFailure) {
 		if (!onSuccess) {
 			ctx.statusElement.error('Internal error: no onSuccess callback provided to lookupCreation()!');
 			return;
 		}
 		ctx.onLookupCreationSuccess = onSuccess;
+		ctx.onLookupCreationFailure = onFailure || emptyFunction;
```

Comment:
Should probably move these up to accomodate returning the failure if `onSuccess` not provided.